regoservice/eval.go

```diff
@@ -213,27 +218,36 @@ func buildAggregatedRego(query *rego.PreparedEvalQuery) (*rego.PreparedEvalQuery
 	aggregation_pkg_val := expr[aggregation_pkg_prop]
 
 	var aggrQuery *rego.PreparedEvalQuery
-
 	if aggregation_pkg_val != nil {
 		aggregation_pkg := aggregation_pkg_val.(string)
+		var err error
+
 		aggrQuery, err = buildBundledRegoForPackage(aggregation_pkg)
+
 		if err != nil {
 			return nil, err
 		}
 	} else {
 		//it's ok skip aggregation package - no aggregation features will be available
-		log.Printf("No aggregation package configured!!!")
+		log.Logger.Infof("No aggregation package configured!!!")
 	}
 	return aggrQuery, nil
 }
 
 func BuildExternalRegoEvaluator(filename string, body string) (data.Inpteval, error) {
 	ctx := context.Background()
+	foundPaths := make([]string, 0)
+
+	for _, path := range commonRegoTemplates {
+		if _, err := os.Stat(commonRegoTemplates[0]); !os.IsNotExist(err) {
```

Comment:
Why do we use `[0]`?
If i understood correctly, we need to check each path.

Reply:
it can be one element or zero